### ISSUE TITLE
wutjs.com is down!

### ISSUE THREAD

**Issue by Ehryk** [EpochFail/name-js#3]
Oh noes! Did the domain name expire? Where's my live demo, yo?

**Comment by carterbancroft** [comment]
Yeah, failed at renewing. Also must have hardcoded the domain or something. The IP on it's host is: 23.92.26.251. The domain will be renewed soon hopefully... glad to know someone cares.
